language-bindings/python/utils/create_lib.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+CUR_DIR=$(cd $(dirname $0) && pwd -P)
+ROOT_DIR=${CUR_DIR}/../../..
+
+WAMR_BUILD_PLATFORM=${WAMR_BUILD_PLATFORM:-"linux"}
+
+cd ${ROOT_DIR}/product-mini/platforms/${WAMR_BUILD_PLATFORM}
+
+mkdir -p build && cd build
+cmake ..
+make -j
+
+cp *libiwasm* ${CUR_DIR}/../src/wamr/libs
```

Comment:
```suggestion
cp libiwasm.so ${CUR_DIR}/../src/wamr/libs
```

Reply:
I was using this for making it generic to other platforms. I can make this wheel Linux-only for now if toy prefer to make it more clear